staging/src/github.com/kubeedge/beehive/pkg/core/context/context_factory.go

```diff
@@ -1,6 +1,7 @@
 package context
 
 import (
+	gcontext "context"
```

Comment:
Keep `context` here?

Reply:
package name is `context`, so need add alias